include/klee/Config/config.h

```diff
@@ -13,4 +13,4 @@
 // limitations under the License.
 
 #define LLVM_VERSION_MAJOR 3
-#define LLVM_VERSION_MINOR 6
```

Comment:
Argh, I see your point now. I thought git would be smart enough to remove these existing commits automatically during merge, but it doesn't. I'll have to fix it first.